CHANGELOG.md

```diff
@@ -0,0 +1,451 @@
+
+# Changelog
+
+## Future Release
+
+Some highlights to be included in the next release:
+
+* Bump to chisel 3.5.4 (https://github.com/chipsalliance/rocket-chip/pull/3105)
+
+* As Scala FIRRTL Compiler (SFC) is being sunset, update chisel2 legacy code to chisel3 (https://github.com/chipsalliance/rocket-chip/pull/3097)
+
+* Removal of `RocketTilesKey` and `RocketCrossingKey` (https://github.com/chipsalliance/rocket-chip/pull/3133)
+
+## Version 1.6.0
+
+10 Oct 2022
+
+### Added
+* Support for Chisel 3.5.3 (https://github.com/chipsalliance/rocket-chip/pull/2931, https://github.com/chipsalliance/rocket-chip/pull/2937, https://github.com/chipsalliance/rocket-chip/pull/2947, https://github.com/chipsalliance/rocket-chip/pull/3005)

```

Comment:
```suggestion
* Bump to Chisel 3.5.3 (https://github.com/chipsalliance/rocket-chip/pull/2931, https://github.com/chipsalliance/rocket-chip/pull/2937, https://github.com/chipsalliance/rocket-chip/pull/2947, https://github.com/chipsalliance/rocket-chip/pull/3005)
```

I don't have any real opinion on what sort of categorization y'all adopt, but I think it's important to call out that any bump of Chisel major versions is a major one for users of rocket-chip. It is a very *good* thing, but it does mean that they will also have to bump Chisel, so it's a big change.